### PR TITLE
feat(rdb): remove endpoints

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by scw-wtimmermans — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  validation: 2024-03-22
+  posted: 2024-03-22
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+When you create a Database Instance, a public endpoint is automatically attributed to it.
+
+For Read Replicas you can choose whether they take on a public or private endpoint.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:19]:
```suggestion
For Read Replicas, you can choose whether they take on a public or private endpoint.
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+If you are using a public endpoint with your Database Instances or Read Replicas, you can remove them anytime in the Scaleway console.
+
+<Macro id="requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:25]:
```suggestion
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:26]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:27]:
```suggestion
- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+</Message>
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:28]:
```suggestion
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+<Message type="tip">
+  Click **Add public endpoint** in the **Network** section to re-add a public endpoint. When you add a new public endpoint, a new IPv4 address and port will be generated. Make sure you update the configuration of your applications with the new endpoint.
+</Message>
+
+## From Read Replicas
+
+<Macro id="requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:51]:
```suggestion
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:53]:
```suggestion
- A [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+</Message>
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:54]:
```suggestion
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+      When you remove a public endpoint your Read Replica becomes inaccessible over the internet.
+    </Message>
+5. Click **Remove public endpoint** to do so.
+
+    Your Read Replica re-initializes.
+
+<Message type="tip">
+  Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will stay the as the previous.
+</Message>
+
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:70]:
```suggestion


```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+<Navigation title="See also">
+  <PreviousButton to="/managed-databases/postgresql-and-mysql/how-to/connect-database-private-network">How to connect a Database Instance to a Private Network</PreviousButton>
+  <NextButton to="/managed-databases/postgresql-and-mysql/how-to/create-read-replica">How to create Read Replicas</NextButton>
+</Navigation>
+
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:75]:
```suggestion
```

**Review by jcirinosclwy — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
+
+## From Database Instances
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll to the **Network** section.
+4. Click **Remove** next to **Public endpoint**. A pop-up appears.
+    <Message type="important">
+      When you remove a public endpoint your Database Instance becomes inaccessible over the internet.
```
Comment (SamyOubouaziz) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:35]:
```suggestion
      When you remove a public endpoint, your Database Instance becomes inaccessible over the internet.
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- A [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll down to the **Read Replica** section.
+4. Click <Icon name="more" /> > **Remove public endpoint**. A pop-up appears.
+    <Message type="important">
+      When you remove a public endpoint your Read Replica becomes inaccessible over the internet.
```
Comment (SamyOubouaziz) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:57]:
```suggestion
      When you remove a public endpoint, your Read Replica becomes inaccessible over the internet.
```

File: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+    </Message>
+5. Click **Remove public endpoint** to do so.
+
+    Your Read Replica re-initializes.
+
+<Message type="tip">
+  Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will stay the as the previous.
```
Comment (SamyOubouaziz) [inline managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx:64]:
```suggestion
  Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will remain the same as the previous one.
```

File: managed-databases/postgresql-and-mysql/how-to/connect-database-private-network.mdx [hunk 14]
```diff
@@ -69,8 +78,15 @@ You can use Public Gateways in [SHH bastion](https://www.scaleway.com/en/docs/ne
 4. Click <Icon name="unlink" /> next to the static IP that you want to unlink from your Database Instance.
 5. Click **Detach resource** to remove the Instance from your Private Network.
 
+<Message type="note">
+This action takes a few moments to complete. During this time, your Database Instance:
+  - remains available
+  - goes into **Configuring** mode
+  - network configuration actions become unavailable
```
Comment (SamyOubouaziz) [inline managed-databases/postgresql-and-mysql/how-to/connect-database-private-network.mdx:85]: The 3rd point doesn't fit in the sentence

**Review by SamyOubouaziz — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/how-to/connect-database-private-network.mdx (hunk 14 above)
Comment (ldecarvalho-doc) [inline managed-databases/postgresql-and-mysql/how-to/connect-database-private-network.mdx:85]:
```suggestion
This action takes a few moments to complete. During this time, your Database Instance:
  - remains available,
  - goes into **Configuring** mode, and
  - network configuration actions become unavailable
```

**Pull request merged**